torchvision/prototype/transforms/_utils.py

```diff
@@ -39,6 +31,20 @@ def get_image_dimensions(image: Union[PIL.Image.Image, torch.Tensor, features.Im
     return channels, height, width
 
 
+def query_image_dimensions(sample: Any) -> Tuple[int, int, int]:
+    flat_sample, _ = tree_flatten(sample)
+    image_dimensionss = {
```

Comment:
Typo:
```suggestion
    image_dimensions = {
```

Reply:
Not quite. `image_dimensionss` is a set of `image_dimensions`. Thus, the extra `s` denotes the plural the same way `images` could be a set of `image`. No strong opinion though.

Reply:
Given we need to avoid the word `image` all together, you could name this `dim_set` and be explit.